src/coreclr/tests/src/JIT/Regression/JitBlue/GitHub_27678/GitHub_27678.il

```diff
@@ -0,0 +1,201 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Do an IL jmp instruction in a method with a big local frame,
```

Comment:
Can you please add more information about the issue here? I would include a code snippet showing the improper usage of `r3` in an epilog.

Reply:
Done.